lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -658,8 +658,11 @@ namespace Js
         this->EnsureClearDebugDocument();
         if (this->debugContext != nullptr)
         {
-            this->debugContext->GetProbeContainer()->UninstallInlineBreakpointProbe(NULL);
-            this->debugContext->GetProbeContainer()->UninstallDebuggerScriptOptionCallback();
+            if(this->debugContext->GetProbeContainer())
```

Comment:
Makes sense ... LGTM

---
In reply to: [104546815](https://github.com/Microsoft/ChakraCore/pull/2641#discussion_r104546815) [](ancestors = 104546815)